pages/404/index.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from 'next/image';
+import error404 from "../../public/images/error404.svg";
```

Comment:
**suggestion:** no need to import

ref: https://nextjs.org/docs/app/building-your-application/optimizing/static-assets

Reply:
This has not been resolved